### DISCUSSION TITLE
How the protocol stack adapts to different wireless transmission rates

### DISCUSSION THREAD

**Discussion by tuopu** [openthread/openthread#9428] [Q&A]
Now it is necessary to use the application protocol stack at 915MHz frequency. Do you only need to open the_ Config_ Radio_ 915MHz_ OQPSK_ Support is set to 1 and openthread_ Config_ Radio_ 2p4ghz_ OQPSK_ Support can be set to 0, except that the wireless driver needs to modify ask and CSMA time parameters.

**Comment by jwhui** [comment]
Thread is currently specified for operation only using IEEE 802.15.4 2.4 GHz O-QPSK PHY.

The sub-GHz support in OpenThread is currently experimental and does not conform to the current Thread Specification. For more information, check out [AN1350: Single-Band Proprietary Sub-GHz Support with OpenThread](https://www.silabs.com/documents/public/application-notes/an1350-openthread-single-band-proprietary-sub-ghz.pdf).

**Reply by tuopu** [reply]
Thank you for your reply
